packages/example/src/middleware.ts

```diff
@@ -0,0 +1,26 @@
+import acceptLanguageParser from 'accept-language-parser';
+import {headers} from 'next/headers';
+import {NextResponse} from 'next/server';
+import type {NextRequest} from 'next/server';
+import i18n from './i18n';
+
+// Somehow not invoked?
```

Comment:
From a quick glance at the example WIP, the middleware doesn't seem to fire unless there is a "pages" folder as well. After that, it fires on both "pages" and "app" folders.

Reply:
Oh cool, thanks for figuring this out! Seems like a bug on the Next.js side.